pose-detection/src/blazepose_tfjs/detector.ts

```diff
@@ -478,9 +518,23 @@ class BlazePoseTfjsDetector implements PoseDetector {
       auxiliaryLandmarksFiltered = this.landmarksSmoothingFilterAuxiliary.apply(
           auxiliaryLandmarksFiltered, this.timestamp, imageSize,
           true /* normalized */, objectScaleROI);
+
+      // Smoothes pose world landmark coordinates to reduce jitter.
+      if (this.worldLandmarksSmoothingFilterActual == null) {
+        this.worldLandmarksSmoothingFilterActual = new KeypointsSmoothingFilter(
+            constants.BLAZEPOSE_WORLD_LANDMARKS_SMOOTHING_CONFIG_ACTUAL);
+      }
+      actualWorldLandmarksFiltered =
+          this.worldLandmarksSmoothingFilterActual.apply(
```

Comment:
Please use NULL (or whatever is used in JS) instead of imageSize and objectScaleROI to avoid bugs as world landmarks filtering doesn't need those parameters.